public/index.html

```diff
@@ -1,9 +1,30 @@
 <html>
   <head>
     <title>タイピングゲーム</title>
-    <link rel="stylesheet" href="/stylesheets/style.css" />
-    <script type="module" src="/javascripts/main.js"></script>
+    <link rel="stylesheet" href="./stylesheets/style.css" />
+    <script type="module" src="./javascripts/main.js"></script>
   </head>
 
-  <body></body>
+  <body>
+    <br />
+    <br />
+    タイピングゲーム
+    <br />
+    <br />
+    <br />
```

Comment:
`<br/>`を多用してスタイリングするのは良くないです。

https://qiita.com/a-takano/items/600e39da898c06d19b32

```suggestion
    <h1>タイピングゲーム</h1>
```